lib/go-atscfg/atscfg.go

```diff
@@ -22,16 +22,36 @@ package atscfg
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"net"
 	"sort"
 	"strconv"
 	"strings"
-	"time"
 
-	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+// ATSConfigFile is all the information necessary to create an ATS config file, including the file name, path, data, and metadata.
```

Comment:
GoDoc should start with the name of the documented symbol

Reply:
Fixed